armi/tests/Godiva.armi.unittest.yaml

```diff
@@ -0,0 +1,24 @@
+metadata:
+  version: 0.2.5
```

Comment:
I believe this has to be:

>   version: uncontrolled

Or the test will start to slide out of date as we bump the ARMI version number.

We always want to run this test against the latest ARMI version.

Reply:
updated to uncontrolled version